data/json/items/comestibles/veggy_dishes.json

```diff
@@ -533,6 +533,25 @@
     "fun": 3,
     "vitamins": [ [ "vitC", 9 ], [ "calcium", 13 ], [ "iron", 8 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "potato_boiled",
+    "name": { "str": "boiled potato", "str_pl": "boiled potatoes" },
+    "weight": "150 g",
+    "flags": [ "EATEN_HOT", "NUTRIENT_OVERRIDE" ],
+    "color": "brown",
+    "spoils_in": "2 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 162,
+    "description": "A boiled potato, the keystone of comfort food.",
+    "price": 75,
+    "price_postapoc": 25,
+    "material": [ "veggy" ],
+    "volume": "250 ml",
+    "fun": 2,
```

Comment:
I don't think plain boiled potatoes are actually enjoyable to eat; I'd set it to 0.
```suggestion
```